### PR TITLE
refactor(model): #CRRE-579 prepare refactor for model

### PR DESCRIPTION
**Pull request by BastienBenard**
## Describe your changes
Preparation of models for the refactoring of the CREE-579 ticket.
The tests will pass in the next PR

## Issue ticket number and link
[CRRE-579](https://jira.support-ent.fr/browse/CRRE-579)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/java/fr/openent/crre/model/OrderRegionComplex.java
Comment (BastienBenard) [inline src/main/java/fr/openent/crre/model/OrderRegionComplex.java:1]: Ces fonctions ne sont plus utilisées.

**Review by alicedraillard — APPROVED** [review]
c bieng bastieng 👍

File: src/main/java/fr/openent/crre/model/FilterModel.java [hunk 1]
```diff
@@ -267,4 +272,51 @@ public FilterModel setIdsOrder(List<Integer> idsOrder) {
...
+    public FilterModel setOrderDescForOrderList(Boolean orderDescForOrderList) {
+        this.orderDescForOrderList = orderDescForOrderList;
+        return this;
+    }
+
+    @Override
+    public FilterModel clone() {
+        try {
+            FilterModel clone = (FilterModel) super.clone();
+            if (this.idsUser != null) clone.idsUser = new ArrayList<>(this.idsUser);
```
Comment (flmartineau) [inline src/main/java/fr/openent/crre/model/FilterModel.java:307]: Autant de ifs ne sert pas à grand chose

Comment (alicedraillard) [inline src/main/java/fr/openent/crre/model/FilterModel.java:307]: 🤡

**Review by qpic-cgi — APPROVED** [review]

**Review by flmartineau — APPROVED** [review]

**Review by JinDouang — APPROVED** [review]

**Pull request merged**